controller/backup_controller.go

```diff
@@ -142,10 +142,18 @@ func (bc *BackupController) handleErr(err error, key interface{}) {
 		return
 	}
 
```

Comment:
It's better to have a comment on _why_ the backup_controller catch the "failed lock" and perform the special logic.